compiler/src/iree/compiler/Codegen/Common/GPU/test/gpu_vector_distribution.mlir

```diff
@@ -17,8 +17,10 @@ func.func @distribute_elementwise_f16(%a: vector<16x16xf16>, %b: vector<16x16xf1
   // CHECK-DAG: %[[A:.*]] = iree_vector_ext.to_simt %{{.*}} : vector<16x16xf16> -> vector<16xf16>
   // CHECK-DAG: %[[D:.*]] = arith.addf %[[DIVD]], %[[A]] fastmath<reassoc,nnan> : vector<16xf16>
   %d = arith.addf %divd, %a fastmath<reassoc,nnan> : vector<16x16xf16>
-  // CHECK: iree_vector_ext.to_simd %[[D]] : vector<16xf16> -> vector<16x16xf16>
-  return %d : vector<16x16xf16>
+  // CHECK-DAG: %[[CMP:.*]] = arith.cmpf ult, %[[D]], %[[ROOT]] : vector<16xf16>
+  %r = "arith.cmpf"(%d, %root) {predicate = 11 : i64} : (vector<16x16xf16>, vector<16x16xf16>) -> vector<16x16xi1>
```

Comment:
Is there a reason for using the generic printed form of the op here?

Reply:
No reason, I pushed the first test that worked for me. I updated it to the custom assembly form instead now.